src/features/create-dao/steps/DAOSummaryField.module.scss

```diff
@@ -0,0 +1,18 @@
+@use '../../../../node_modules/@zero-tech/zui/styles/colors';
+
+.Field {
+	display: flex;
+	flex-direction: column;
+
+	.FieldLabel {
+		font-size: 0.75rem;
+		color: #{colors.$grey-lighter-2};
```

Comment:
I could be wrong, but I have a feeling this should throw an error ?? 
```
color: $grey-lighter-2;
```

Reply:
Yeah, unsure why this wasn't erroring. Updated!